pybikes/fsm.py

```diff
@@ -0,0 +1,51 @@
+# -*- coding: utf-8 -*-
+# Copyright (C) 2017, aronsky <aronsky@gmail.com>
+
+from lxml import etree
+
+from pybikes.base import BikeShareSystem, BikeShareStation
+from pybikes import utils
+
+
+class FSMSystem(BikeShareSystem):
+    sync = True
+    unifeed = True
```

Comment:
This escaped me before, `unifeed` attribute is only needed for systems that with one call produce more than one instance and is just meant for a reference on code using `pybikes` to know that it should use a cache. Must either be deleted or set to `False`.

My bad, this should be documented.